Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -565,6 +565,8 @@ std::vector<EventsSearchResult> EventsRefactorer::ReplaceStringInEvents(
     bool inActions,
     bool inEventStrings) {
   vector<EventsSearchResult> modifiedEvents;
+  if (toReplace.empty()) return modifiedEvents;

```

Comment:
Should it test newString.empty() too ? (I wonder if replacing a string with an empty one is a use case?)

Reply:
I use this in text editors to remove things.